pkg/kubelet/rkt/rkt.go

```diff
@@ -94,6 +94,10 @@ const (
 	// hence, setting ndots to be 5.
 	// TODO(yifan): Move this and dockertools.ndotsDNSOption to a common package.
 	defaultDNSOption = "ndots:5"
+
+	// Annotations for the ENTRYPOINT and CMD for an ACI that's converted from Docker image.
+	appcDockerEntrypoint = "appc.io/docker/entrypoint"
+	appcDockerCmd        = "appc.io/docker/cmd"
```

Comment:
@yujuhong Thanks, I filed an issue at docker2aci https://github.com/appc/docker2aci/issues/133 Also added a TODO.
Good news is, we don't have to deal with godep right now :)